packages/core/src/shared/clients/s3Client.ts

```diff
@@ -626,6 +636,28 @@ export class DefaultS3Client {
             }
         }
     }
+
+    /**
+     * Gets an object from a bucket.
+     *
+     * The bucket should reside in the same region as the one configured for the client.
```

Comment:
Why do we plan to enforce limitation? Can't we do cross region calls? 
https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/s3-cross-region.html

Reply:
The comment was taken from similar functions. GetObject should be able to get it cross-region.

Reply:
Removed the comment.